spinalcordtoolbox/scripts/sct_dmri_moco.py

```diff
@@ -37,7 +37,7 @@
 def get_parser():
 
     # initialize parameters
-    param_default = ParamMoco(is_diffusion=True, group_size=3, metric='MI', smooth='1')
+    param_default = ParamMoco(is_diffusion=True, group_size=3, metric='MI', smooth='1', iter=10)
```

Comment:
I'm not sure we need this line here, since `ParamMoco` already has the default value in its constructor. (I am happy to make this change on your behalf.)